### PR TITLE
New changes

### PR DESCRIPTION
**Pull request by Neustradamus**
New changes (XMPP, OMEMO, OTR, clients list, links)

### CODE REVIEW

File: pages/chat/clients/ru.text [hunk 1]
```diff
@@ -15,22 +17,30 @@ h2. Рекомендуемые клиенты
...
-| [[Adium]] | macOS | *Да* | Нет | Нет | *Да* | *Да* | Частично | Open source. Lack of updates cause for security concern. DNS and URL hovering information leaks. |
+| [[Adium]] | macOS | *Да* | *Да* | Нет | *Да* | *Да* | Частично | Open source. Lack of updates cause for security concern. DNS and URL hovering information leaks. |
+| [[Jitsi -> https://jitsi.org]] | GNU/Linux, Windows, macOS | *Да* | Нет | *Да* | ? | *Да* | ? | Open source. Написан на Java. |
+| [[Miranda -> https://miranda-im.org]] | Windows | *Да* | Нет | Нет | ? | ? | ? | Open source. Стабильный клиент со множеством плагинов. |
+| [[Miranda NG -> https://miranda-ng.org]] | Windows | *Да* | Нет | Нет | ? | ? | ? | Open source. Стабильный клиент со множеством плагинов. |
+| [[Pidgin]] | GNU/Linux, Windows | *Да* | *Да* | *Да* (Linux) | *Да* | *Да* | *Да* | Open source. Стабильный мессенджер со множеством функций. *Обязательно используйте самую свежую версию!* |
+| [[Spark -> https://www.igniterealtime.org/projects/spark/]] | GNU/Linux, Windows, macOS | *Да* | Нет | *Да* | ? | ? | ? | Open source. Написан на Java. |
+
+h2. Клиенты чтобы избежать
```
Comment (tehnick) [inline pages/chat/clients/ru.text]: This is incorrect phrase in Russian.

**Comment by tehnick** [comment]
Too many changes for one commit. It is difficult to understand the purpose some of them. Description is too short and not informative.

File: pages/chat/clients/ru.text (hunk 1 above)
Comment (Neustradamus) [reply pages/chat/clients/ru.text]: What is the good?

Comment (Ri0n) [reply pages/chat/clients/ru.text]: It's rather like "Неполноценные клиенты" or "Малосовместимые клиенты" or maybe "Клиенты, которых стоит избегать"

Comment (tehnick) [reply pages/chat/clients/ru.text]: "Клиенты, которых стоит избегать"

**Comment by Neustradamus** [comment]
@elijh: Good for you?

**Comment by Neustradamus** [comment]
@elijh: Have you looked?

**Comment by disturbio** [comment]
Pidgin and Coyim really the only XMPP client that we have been supporting in practice. I'm ok with removing it, but also unsure about how good is the idea to recommend Psi and/or Psi+ / Conversations and/or Conversations Legacy. 

I have 0 idea about Psi, but if it's an ok client and it makes worth to replace pidgin having two of them is confusing. Conversations seems like the go-to client on Android, but I don't have an Android device which means I have to request one from another person. Apart from the confusion from two "Conversations", it means I have more time taking another's persons device to do any kind of test regarding that server which is something i would prefer to avoid.

**Comment by tehnick** [comment]
@Neustradamus is one of the most active user of Psi+ during very long period of time, so his opinion about Psi and Psi+ may be far from objective.

Trying be more objective (it is hard because I am one of Psi/Psi+ developers) I would say that currently the most convenient and secure XMPP clients for regular users are Gajim (for desktop) and Conversations (for smartphones).

As for Psi and Psi+ -- they are oriented to experienced XMPP users who need many options for program customization (as accented at their websites), but have being developing less active than Gajim and currently lack one important modern feature: [MAM](https://xmpp.org/extensions/xep-0313.html).

Personally I do not agree with changes in this PR, but this is your website and your opinion is the most important here.

**Comment by Neustradamus** [comment]
@disturbio: Tables are good.

For desktop:
- Pidgin is not a full perfect XMPP client (not new XMPP RFCs, missing XEPs, and not up-to-date XEPs, OMEMO is not integrated by default, ...), of course same for Adium.
- CoyIM has not OMEMO, only OTR.
- Psi and Psi+ has perfect XMPP client like Gajim.

For mobile:
- Conversations has now OMEMO instead of OTR : https://play.google.com/store/apps/details?id=eu.siacs.conversations
- Conversations Legacy is the old Conversations with OTR : https://play.google.com/store/apps/details?id=eu.siacs.conversations.legacy
- ChatSecure
- Zom

**Comment by Neustradamus** [comment]
Some days after ^^

**Comment by Neustradamus** [comment]
@elijh: Can you confirm this PR?

Last changes:
- Gajim 0.16.9 is the last build with OTR, since 1.0.0 it is only OMEMO
- Adium has OTR included, not OMEMO (external plugin)
- Pidgin has OTR and OMEMO with external plugins

Please note that:
- Psi and Psi+ have OTR and OMEMO with integrated plugins.

**Comment by elijh** [comment]
Deprecating pidgin is long overdue! I am fine with elevating Psi. Never used it, but it has a solid rep.

**Comment by Neustradamus** [comment]
@elijh: Thanks!

**Pull request merged**